WordPress/gutenberg
# Scripts: Update eslint-plugin-markdown version

## Description

A minor fix to bump the version of eslint-plugin-markdown module.
This fixes an audit warning for the trim module.

## How has this been tested?

- From packages/scripts run `npm install --package-lock-only`
- Run `npm audit` confirm warning on trim
- Apply PR and confirm warning is removed
- Confirm no regression by using: `npm run lint:md-docs`

The lint command should still run, it will show warnings that means it is working


## Types of changes

Dependency update.
